SEED-platform/seed
# #3308 added audit template building id

#### Any background context you want to provide?
I wanted the Audit Template Building ID to be included in the import of BuildingSync files from Audit Template Tool.

#### What's this PR do?
Add Audit Template Building ID and Portfolio Manager Building ID to the import of BuildingSync files.

#### How should this be manually tested?
Import the sample [BuildingSync file](https://drive.google.com/file/d/1Ra6Y9q45WsPPNlB6nTxHFRUWIBLUd9BY/view?usp=sharing) and you should see the Audit Template Building ID like that indicated in the image below.

#### What are the relevant tickets?
#3308

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/6326294/174097267-7ab14e4f-d9fc-41d0-91e0-1583740e1ef4.png)
